apps/multisig-signer/src/app/components/remove-signer-form/remove-signer-form.tsx

```diff
@@ -0,0 +1,121 @@
+import { useState } from 'react';
+import { gql, useLazyQuery } from '@apollo/client';
+import { captureException } from '@sentry/react';
+import { t } from '@vegaprotocol/react-helpers';
+import { useEthereumTransaction } from '@vegaprotocol/web3';
+import {
+  FormGroup,
+  Input,
+  Button,
+  InputError,
+  Loader,
+} from '@vegaprotocol/ui-toolkit';
+import { prepend0x } from '@vegaprotocol/smart-contracts';
+import { useContracts } from '../../config/contracts/contracts-context';
+import type { FormEvent } from 'react';
+import type {
+  RemoveSignerBundle,
+  RemoveSignerBundleVariables,
+} from '../__generated__/RemoveSignerBundle';
+import type { MultisigControl } from '@vegaprotocol/smart-contracts';
+
+const REMOVE_SIGNER_QUERY = gql`
+  query RemoveSignerBundle($nodeId: ID!) {
+    erc20MultiSigSignerRemovedBundles(nodeId: $nodeId) {
+      edges {
+        node {
+          oldSigner
+          nonce
+          signatures
+        }
+      }
+    }
+  }
+`;
+
+export const RemoveSignerForm = () => {
+  const { multisig } = useContracts();
+  const [address, setAddress] = useState('');
+  const [bundleNotFound, setBundleNotFound] = useState(false);
+  const [runQuery, { data, error, loading }] = useLazyQuery<
+    RemoveSignerBundle,
+    RemoveSignerBundleVariables
+  >(REMOVE_SIGNER_QUERY);
+  const { perform, Dialog } = useEthereumTransaction<
+    MultisigControl,
+    'remove_signer'
+  >(multisig, 'remove_signer');
+  const handleSubmit = async (e: FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    setBundleNotFound(false);
+    try {
+      if (address === '') {
```

Comment:
suggestion (non blocking): could just use react hook form here to get some easier client side validation. Deffo follow up though

Reply:
https://github.com/vegaprotocol/frontend-monorepo/issues/1559